src/math-history.c

```diff
@@ -54,40 +54,25 @@ scroll_bottom_cb(MathHistory *history, gpointer data)
 void
 math_history_insert_entry (MathHistory *history, char *equation, MPNumber *answer, int number_base)
 {
```

Comment:
There is a new build warning.
```
math-history.c:55:88: warning: unused parameter 'number_base' [-Wunused-parameter]

math_history_insert_entry (MathHistory *history, char *equation, MPNumber *answer, int number_base)
```
https://travis-ci.org/github/mate-desktop/mate-calc/jobs/759745686#L1704